community/btpd/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+autoreconf --verbose --install --force
```

Comment:
We don't run `autoreconf` unless necessary. Does it work when this is omitted?

Reply:
Wow. This project is really dead. Guess we need to keep this. Change it to: `autoreconf -fis`

Reply:
Don't work without autoreconf.

Reply:
Yup. Change it to `autoreconf -fis`.

Reply:
Changed.